src/OpenLoco/Industry.cpp

```diff
@@ -251,15 +236,14 @@ namespace OpenLoco
                     const auto* industryObject = tileIndustry->getObject();
                     if (industryObject != nullptr)
                     {
-                        auto animOffsets = word_4F9274;
+                        auto animOffsets = getUnk4F9274(false);
                         if (industryObject->var_C6 & (1 << industryEl->var_6_1F()))
```

Comment:
You can just call the function from here getting the type from this

Reply:
I'm not sure I understand. Are you suggesting just moving `getUnk4F9274(false)` into the `if` block or send the value of the `&` check as the type for the function?

Reply:
Do `getUnk4F9274(industryObject->var_C6 & (1 << industryEl->var_6_1F());`  you'll need to double check which way round it is.